src/main/kotlin/no/skatteetaten/aurora/boober/service/AuroraDeploymentContextService.kt

```diff
@@ -278,6 +279,26 @@ class AuroraDeploymentContextService(
             "Both Webseal-route and OpenShift-Route generated for application. If your application relies on WebSeal security this can be harmful! Set webseal/strict to false to remove this warning."
         } else null
 
+        val azureManagedRoute = features.filter { (feature, spec) ->
+            feature is AzureFeature && feature.isActive(spec)
+        }.isNotEmpty()
```

Comment:
```suggestion
        val azureManagedRoute = features.any { (feature, spec) ->
            feature is AzureFeature && feature.isActive(spec)
        }
```
Dett kan for øvrig også gjøres andre steder i denne filen.